### PR TITLE
(1) Confusing narrative in version control;  (2) typos in shell.

### PR DESCRIPTION
**Pull request by SteveGoldstein**
I had to puzzle over some of the text in github/Readme to figure out what it meant.  Maybe it's more clear in the context of the workshop, but I've suggested edits to help it as stand-alone text.

### CODE REVIEW

**Comment by gonuke** [comment]
@gidden should peak at the 2nd commit here on Git remote

**Comment by gidden** [comment]
I'll tackle this.

On Tue, Jan 13, 2015 at 12:45 PM, Paul Wilson notifications@github.com
wrote:

> @gidden https://github.com/gidden should peak at the 2nd commit here on
> Git remote
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/UW-Madison-ACI/boot-camps/pull/161#issuecomment-69786280
> .

## 

Matthew Gidden
Ph.D. Candidate, Nuclear Engineering
The University of Wisconsin -- Madison
Ph. 225.892.3192

File: version-control/git/github/Readme.md [hunk 1]
```diff
@@ -44,8 +44,38 @@ you want an
...
+
+A fork is a copy of a repository. Forking a repository allows you to freely experiment with changes without affecting the original project.
+
+or 
+
+From https://github.com/octocat/Spoon-Knife
+
+Creating a fork is producing a personal copy of someone else's project. A fork acts as a bridge between the original repository and your personal copy. You can submit Pull Requests to help make other people's projects better by offering your changes to the original project. Forking is at the core of social coding at GitHub.
+
+-->
```
Comment (gidden) [inline version-control/git/github/Readme.md]: I agree that a sentence or two about what forking is would be useful.

File: version-control/git/github/Readme.md [hunk 2]
```diff
@@ -44,8 +44,38 @@ you want an
...
+This isn't clear.  The reader does not yet know what "adding the original as a remote repository" means. 
+
+I think this needs to be explicit, like: 
+1. Forking creates a copy of the repo on the servers.
+2. Cloning makes a local copy of that repo on your computer.
+
+That local copy is connected to your fork.  Now you need to connect that fork 
+to the original repository.  Here are the steps...
+
+-->
```
Comment (gidden) [inline version-control/git/github/Readme.md]: Also agreed, this language should be tightened up.

File: version-control/git/github/Readme.md [hunk 3]
```diff
@@ -96,14 +126,17 @@ Step 3 : Add a connection to the common repository :
     upstream        https://github.com/UW-Madison-ACI/simplestats (fetch)
     upstream        https://github.com/UW-Madison-ACI/simplestats (push)
 
+<!-- This is an aside.  Maybe it should be labeled "Naming convention" and in 
+smaller font.
+-->
```
Comment (gidden) [inline version-control/git/github/Readme.md]: sure

File: version-control/git/github/instructor_notes.md [hunk 4]
```diff
@@ -1,5 +1,13 @@
 # git/GitHub
 
+<!-- 
+
+This file looks like needs to be updated to correspond to the current order of the 
+curriculum.   We now introduce github after "local", "Make Incremental Changes 
+I and II."  This introduces github at the beginning.
+
+-->
```
Comment (gidden) [inline version-control/git/github/instructor_notes.md]: I don't think I've ever looked at/touched the instructor notes (for better or for worse...); feel free to update them if you'd like to!

**Comment by gidden** [comment]
Hi @SteveGoldstein, I agree with your comments. Would you like to go ahead and propose concrete edits and we can get them merged in? Thanks!

**Comment by SteveGoldstein** [comment]
Yes, I'll work on this doc either tonight or during tomorrow's 
workshop.   (But I'm a markdown newbie and might need some help with 
that part).

Steve

**Comment by gidden** [comment]
I'm happy to help with whatever questions you have.

**Pull request closed without merging**